scripts/mac_intel_install.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/zsh
+;#!/bin/zsh
```

Comment:
I think you need this - potentially it should be bash if you're worried about people not having zsh?

Reply:
Ah you're right

Reply:
Removed